kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/main/java/org/kie/server/springboot/samples/WebSecurityConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.kie.server.springboot.samples;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration("kieServerSecurity")
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+        .csrf().disable()
+        .authorizeRequests()
+            .antMatchers("/rest/server*").authenticated()
+            .and()
+        .httpBasic();
+    }
+
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+        auth.inMemoryAuthentication().withUser("kieserver").password("kieserver1!").roles("kie-server");        
+        auth.inMemoryAuthentication().withUser("john").password("john@pwd1").roles("kie-server", "PM", "HR");        
```

Comment:
Should probably match what gets generated in the archetype?

auth.inMemoryAuthentication().withUser("user").password("user").roles("kie-server");
auth.inMemoryAuthentication().withUser("kieserver").password("kieserver1!").roles("kie-server");

Reply:
Don’t think we need to align it that much as this class is usually replaced